custom_components/abbfreeathome_ci/valve.py

```diff
@@ -0,0 +1,100 @@
+"""Create ABB-free@home valve entities."""
+
+from typing import Any
+
+from abbfreeathome.devices.heating_actuator import HeatingActuator
+from abbfreeathome.freeathome import FreeAtHome
+
+from homeassistant.components.valve import (
+    ValveDeviceClass,
+    ValveEntity,
+    ValveEntityDescription,
+    ValveEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import CONF_SERIAL, DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up valves."""
+    free_at_home: FreeAtHome = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities(
+        FreeAtHomeValveEntity(valve, sysap_serial_number=entry.data[CONF_SERIAL])
+        for valve in free_at_home.get_devices_by_class(device_class=HeatingActuator)
+    )
+
+
+class FreeAtHomeValveEntity(ValveEntity):
+    """Defines a free@home valve entity."""
+
+    _attr_should_poll: bool = False
+
+    def __init__(self, valve: HeatingActuator, sysap_serial_number: str) -> None:
+        """Initialize the valve."""
+        super().__init__()
+        self._valve = valve
+        self._sysap_serial_number = sysap_serial_number
+
+        self.entity_description = ValveEntityDescription(
+            key="HeatingActuatorValve",
+            device_class=ValveDeviceClass.WATER,
+            name=valve.channel_name,
+            reports_position=True,
+        )
```

Comment:
Sorry, I missed this the first time. Anytime I can specify a property in the entity_description I do, this ensure's we don't always overwrite the Home Assistant property code and allows for the HomeAssistant to apply it's own logic.

```suggestion
        self.entity_description = ValveEntityDescription(
            key="HeatingActuatorValve",
            device_class=ValveDeviceClass.WATER,
            entity_registry_enabled_default=False,
            name=valve.channel_name,
            reports_position=True,
        )
```